modules/serverless-access-custom-domain.adoc

```diff
@@ -12,15 +12,23 @@
 
 [source,terminal]
 ----
-$ curl -H "Host: custom-ksvc-domain.example.com" http://<ip_address>
+$ curl -H "Host: custom-ksvc-domain.example.com" \
+    http://<ingress-router-IP-or-hostname>
 ----
 
-+
-where `<ip_address>` is the IP address that the {product-title} ingress router is exposed to.
-+
-
 .Example output
 [source,terminal]
 ----
 Hello OpenShift!
 ----
+
+[NOTE]
```

Comment:
The previous format for providing info about the ip address was correct. I think that obtaining the ingress IP or hostname should be the first step, something like:

```
. Obtain the ingress IP address or host name for the ingress router. This process varies depending on the platform that hosts the {product-title} cluster.
+
For example, to find the IP address for GCP or Azure:
+
[source,terminal]
----
$ oc get svc -n openshift-ingress router-default -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
----
```

And that this note should be removed.

Reply:
Note removed